src/test/config/kind/common-values.yaml

```diff
@@ -46,3 +46,7 @@ ingress:
   create: true
   host: localhost
   https: false
+
+monitoring:
```

Comment:
Can we also add the values to the `expected_helm_output` or do you think this is sufficient?

Reply:
KinD checks the actual metrics, but yeah, we can enable jmx to have more stuff in expected output too.

Reply:
Enabled jmx monitoring for helm output test and test config used to deploy to EKS